transposon/genotypeClones.py

```diff
@@ -510,8 +496,9 @@ def postCleanUp(G, annotation, maxRate):
     parser.add_argument('--minCentrality', action='store', type=float, default=0.2, help='Each BC-cell edge must represent at least this proportion of UMIs for BC')
     parser.add_argument('--maxpropreads', action='store', type=int, default=0.1, help='Edges joining communities must have fewer than this number of UMIs as proportion of the smaller community they bridge')
 
-    parser.add_argument("--splitConflictingClones", action="store", type=str, default=None, help="Run a post-processing clean-up and remove conflicting cells based on BCs annotation.")
-    parser.add_argument("--maxConflictRate", action="store", type=float, default=0.0, help="Maximum non-majority transfection UMI rate to remove a conflicting cells")
+    parser.add_argument("--transfectionKey", action="store", type=str, default=None, help="Attribute key used to identify BCs transfections")
+    parser.add_argument("--maxConflictRate", action="store", type=float, default=None, help="Maximum non-majority transfection UMI rate to be remove from conflicting cells. It requires `transfectionKey` to execute")
+    parser.add_argument("--removeConflictingCells", action='store_true', default=False, help = "Removes conflict clones after all clean-up. It requires `transfectionKey` to execute")
```

Comment:
"Remove cells linked to BCs from conflicting transfections"?

Reply:
It supposes to be "remove cells linked to BCs from 2+ transfections", I've updated the help to reflect it